wireguard-server/wg-wirescale.conf

```diff
@@ -0,0 +1,5 @@
+[Interface]
+PrivateKey = <server_private_key>
+Address = 10.8.0.1/24
+ListenPort = 51820
+SaveConfig = true
```

Comment:
Setting `SaveConfig` to true and doing `wg-quick save wg-wirescale` looks to be equivalent 🤔 
If so, should we remove one?

Reply:
You're right -- the `wg-quick save wg-wirescale` hack is currently used specifically because this configuration doesn't have any effect. Would be nice with a cleaner solution, but I've removed `SaveConfig = true` for now :)